earth_enterprise/src/common/tests/RunAllTests.pl

```diff
@@ -17,12 +17,24 @@
 
 use FindBin qw($Bin $Script);
 use File::Basename;
+use Cwd;
+use Cwd 'abs_path';
+
+# This script should be run from NATIVE-*-x86_64/bin/tests
+my $workingDir = abs_path(cwd());
+my $scriptDir = dirname(abs_path($0));
+my $notInBuildDir = (index($scriptDir, "NATIVE") == -1);
+if ($workingDir ne $scriptDir || $notInBuildDir) {
```

Comment:
Minor: Instead of testing `$workingDir ne $scriptDir` you could also do:

```Perl
chdir($scriptDir);
```

That way the script would work from anywhere, as long as it's called from a build directory.

In total it would be something like:

```Perl
# This script should be run from NATIVE-*-x86_64/bin/tests
my $scriptDir = dirname(abs_path($0));
chdir($scriptDir);
if ($scriptDir !~ /NATIVE-[a-zA-Z0-9_-]*\/bin\/tests$/) {
# . . .
}
```